pkg/blob/downloader.go

```diff
@@ -57,6 +57,10 @@ func (d *downloader) Download(ctx context.Context, pathOrURI string) (Blob, erro
 			path, err = paths.URIToFilePath(pathOrURI)
 		case "http", "https":
 			path, err = d.handleHTTP(ctx, pathOrURI)
+			if err != nil {
+				// retry
```

Comment:
```suggestion
				// retry as we sometimes see `wsarecv: An existing connection was forcibly closed by the remote host.` on Windows
```

Reply:
It's too bad we don't have greater insight into why this happens - but this seems sensible for now